### PR TITLE
Clean code and enhance readme.md #RB-548

### PR DESCRIPTION
**Pull request by hugifix**
Try to clean a little bit the code and improve the readme.md. If you see more thinks to clean please feel free to participate.

### CODE REVIEW

File: requirements-grid.html [hunk 1]
```diff
@@ -113,7 +111,7 @@
               <time is="relative-time" class="time"
                     datetime$="[[item.creationDate]]">
               </time>
-              <div class="card-content" class="truncate">{{item.description}}</div>
+              <div class="card-content" truncate>{{item.description}}</div>
```
Comment (istvank) [inline requirements-grid.html]: What was wrong here?

Comment (hugifix) [reply requirements-grid.html]: 2 times class attribute for the div element. This is not allowed I guess.

Comment (istvank) [reply requirements-grid.html]: Sure, but then it should be added within the existing class attribute.

Comment (hugifix) [reply requirements-grid.html]: Oh yeah .... thanks

Comment (hugifix) [reply requirements-grid.html]: Fixed

**Comment by hugifix** [comment]
Any more insight? You want to change or clean something? If not, you can merge.

**Pull request merged**